python/python/Ice/ObjectPrx.py

```diff
@@ -38,6 +153,27 @@ def ice_isA(self, id, context=None):
         else:
             return super().ice_isA(id, context)
 
+    def ice_isAAsync(self, id, context=None):
+        """
+        Test whether this object supports a specific Slice interface.
+
+        Parameters
+        ----------
+        id : str
+            The type ID of the Slice interface to test against.
+        context : dict, optional
+            The context dictionary for the invocation.
+
+        Returns
+        -------
+        bool
```

Comment:
Not sure how this is usually documented in Python but doesn't this really return a future that will contain a bool once it completes?